pkg/reconcilermanager/controllers/util.go

```diff
@@ -271,6 +272,12 @@ func ociSyncEnvs(opts ociOptions) []corev1.EnvVar {
 		Name:  reconcilermanager.OciSyncWait,
 		Value: fmt.Sprintf("%f", opts.period),
 	})
+	if useCACert(opts.caCertSecretRef) {
+		result = append(result, corev1.EnvVar{
+			Name:  "SSL_CERT_FILE",
```

Comment:
Should we use TLS instead of SSL here or is it SSL for some particular reason?

Reply:
The env var name here is consumed directly by the [crypto library](https://pkg.go.dev/crypto/x509#SystemCertPool)

Reply:
Can you pull this out to a const with a description?
Maybe link to the internal const https://cs.opensource.google/go/go/+/master:src/crypto/x509/root_unix.go;l=19

Reply:
Done